{{cookiecutter.repo_name}}/scripts/setup_js.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
 npm install
+npm run build
```

Comment:
Should this be run during the setup? Its omission doesn't prevent someone from running the dev server, and since it needs to be rerun any time a change is made, it might be better not to have an old build lying around by default that could potentially lead to confusion

Reply:
Its omission **does** prevent certain tests from passing with this change though, which is why I'm adding it. This change means a build doesn't actually always happen when tests run, which means there needs to be some initial build to make sure tests using the Django client pass. Is there more I could add to the docs to explain this better?

Reply:
Got it, that makes sense. Thanks for the detail/explanation